lib/vdsm/storage/lvm.py

```diff
@@ -1384,24 +1384,25 @@ def removeVGbyUUID(vgUUID):
         removeVG(vg.name)
```

Comment:
The commit message suggests that this change add missing tests, but the actual change looks like change in behavior, like acccepting vg uuid instead of vg name, and doing a look up of vg by UUID.

Please add the missing tests before changing the behavior to make sure we don't break callers.